src/main/java/db/migration/V2__migrations_from_2_5.java

```diff
@@ -47,6 +47,49 @@ public Integer getChecksum() {
     public void migrate(Connection connection) throws Exception {
         this.connection = connection;
         removeClient("auth-server");
+        addDisplayToMultiValuedAttributes();
+    }
+
+    private void addDisplayToMultiValuedAttributes() throws SQLException {
```

Comment:
Alright. Maybe we should migrate to a Spring JDBC migration, to make this less complex, but that should be done in an upcoming PR.